modules/user_accounts/php/NDB_Form_user_accounts.class.inc

```diff
@@ -2,7 +2,7 @@
 /**
 * The user account page
 *
-* PHP Version 5
+* PHP Version 7
```

Comment:
Isn't PHP 7 the minimum version we support? My motivation was to update the documentation accordingly.